src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -372,59 +362,59 @@
 
         "deletedLength": 
         {
-          "description": "An optional integer specifying the number of bytes to delete, start at the byte offset specified by the offset property, measured from the beginning of the file.",
+          "description": "Specifies the number of bytes to delete, starting at the byte offset specified by offset, measured from the beginning of the file.",
           "type": "integer",
           "minimum": 1
         },
 
         "insertedBytes": 
         {
-          "description": "An optional string that specifies the byte sequence to be inserted at the byte offset specified by the offset property, measured from the beginning of the file.",
+          "description": "An optional string that specifies the byte sequence to be inserted at the byte offset specified by the 'offset' property, measured from the beginning of the file.",
           "type": "string"
         }
       },
-      "required": ["offset", "deletedLength", "insertedBytes"]
+      "required": ["offset"]
```

Comment:
JSchema can't yet parse `anyOf`, so I filed a issue lgolding/jschema#67 to do it, and #179 in sarif-sdk to scrub the schema for opportunities to use it.